tests/basic/pool_function_test.go

```diff
@@ -833,3 +834,87 @@ var _ = Describe("{PoolExpandAddDiskInMaintenanceMode}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
+
+	var testrailID = 34542876
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542876
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolVolUpdateResizeDisk", "expand volume to the pool and pool expansion using resize-disk", nil, testrailID)
```

Comment:
nit: this test description can be improved with clarity.

Reply:
changed it.